test/image/mocks/table.json

```diff
@@ -0,0 +1,114 @@
+{
+    "layout": {
+        "width": 1200,
+        "height": 800,
+        "title": "Widget parameters and cost",
+        "font": {
+            "color": "blue",
+            "size": 200,
+            "family": "sans-serif"
+        },
+        "margin": {"t": 106, "r": 60, "b": 54, "l": 60}
+    },
+
+    "data": [{
+
+        "type": "table",
+
+        "domain": {
+            "x": [0.05, 0.95],
+            "y": [0.02, 0.97]
+        },
+
+        "columnwidth": [110, 60, 90, 90, 80, 70, 80, 60, 100, 70],
+
+        "header": {
+
+            "values": [
+                ["Height"],
+                ["Width"],
+                ["Cylinder"],
+                ["Weight"],
+                ["Cost"],
+                ["Penalty"],
+                ["H/W"],
+                ["Min H/W"],
+                ["Diameter"],
+                ["RF block"]
+            ],
```

Comment:
Nope, it handles a 1D array just fine, I'll need to update the mock, thanks for the reminder!